using_images/other_images/jenkins.adoc

```diff
@@ -417,3 +417,25 @@ Jenkins job] that is pre-configured in the Jenkins image utilizes the
 plug-in for creating CI/CD flows for {product-title} in Jenkins.
 
 See the https://github.com/openshift/jenkins-plugin/[the plug-in's README] for a detailed description of what is available.
+
+== OpenShift Client Plug-in
+
+The experiences gained working with users of the OpenShift Pipeline plug-in, coupled with the rapid evolution of both Jenkins
+and OpenShift, have provided valuable insight into how to best provide {product-title} integration from Jenkins jobs.
+
+As such, the new experimental https://github.com/openshift/jenkins-client-plugin[OpenShift Client Plugin for Jenkins]
+is now offered as a technical preview and is included in the OpenShift Jenkins images on Centos (docker.io/openshift/jenkins-1-centos7:latest
+and docker.io/openshift/jenkins-2-centos7:latest). The plugin is also available from the Jenkins Update Center.  Envisioned as the long term
```

Comment:
*docker.io/openshift/jenkins-1-centos7:latest* and *docker.io/openshift/jenkins-2-centos7:latest*